public/app/project-setup/_components-theme.scss

```diff
@@ -0,0 +1,16 @@
+// this file contains all imports needed for components
+// but no global definitions
+@import "foundation/functions";
+
+@include exports("global") {
```

Comment:
Why do we use this notation, we have varaible in foundation, or is doesn't work?